runtime/doc/starting.txt

```diff
@@ -453,8 +453,9 @@ accordingly, proceeding as follows:
 	set or when using $VIMINIT.
 
      c. If the 'exrc' option is on (which is NOT the default), the current
-	directory is searched for two files.  The first that exists is used,
+	directory is searched for three files.  The first that exists is used,
 	the others are ignored.
```

Comment:
Should we show an error if more than one is found? Else we should define an explicit precedence, not just alphanumeric?

Reply:
> Should we show an error if more than one is found?

That would require to search for all of them? Currently the next one is search only if the previous one is not found.

> we should define an explicit precedence, not just alphanumeric?

The order is actually hardcoded:
- .nvimrc.lua
- .nvimrc
- .exrc

Should we change anything here?

Reply:
Seems ok, let's leave it.